src/main/webapp/scripts/group-script.js

```diff
@@ -24,6 +22,7 @@ var searchName;
 var searchMemberKey;
 var orderBy;
 var viewTotal = 200;
+var showOnlyParentGroups = false;
```

Comment:
Maybe showOnlyParentGroups could be the default option and this could be a flatten option which will show all groups as individual circles, that way neither option has groups repeated on the screen.

Reply:
Sounds good